sqlitecluster/SQLiteNode.cpp

```diff
@@ -580,6 +580,7 @@ bool SQLiteNode::update() {
                 peer->erase("StandupResponse");
             }
             _changeState(STANDINGUP);
+            _leaderVersion = _version;
```

Comment:
I feel like this should get rest to blank somewhere when we stop leading. It seems it currently *doesn't*, but this seems like as good a time as any to investigate. I worry that if we stop leading, but nobody else is leading yet, this value may be wrong. This may have implications that I haven't fully thought through for the status check we're changing as well.

Reply:
That makes sense. I don't think we want to set it to blank while `STANDINGDOWN` because that is still needed for this change. I think this is the right place to set `_leaderVersion = ""`

https://github.com/Expensify/Bedrock/blob/7fc389f94237c72591f909fba622df1e79c6a0d9/sqlitecluster/SQLiteNode.cpp#L958-L959

Do you think that is sufficient? Or would it be better to remove this change and rewrite the conditional like it was and only check leader version if `FOLLOWING`?

Reply:
I think the right place is probably in here: https://github.com/Expensify/Bedrock/blob/7fc389f94237c72591f909fba622df1e79c6a0d9/sqlitecluster/SQLiteNode.cpp#L1777

If the existing state is `LEADING` or `STANDINGDOWN` and then new state is anything else.

Reply:
Ok, I added it there

Reply:
I actually don't think we can set this at `STANDINGUP`, I think we need to wait until `LEADING`.

Reply:
That makes sense - change pushed.